README.md

```diff
@@ -1,59 +1,28 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse!
 
-> One paragraph statement about the project.
+> This is my first project at Micoverse. In it I include Linterns to check syntax errors, potencial problems and check the standard adherance of the code .
 
-![screenshot](./app_screenshot.png)
+![screenshot](./screenshot-01042021.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- Css
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
```

Comment:
- [ ] Please, set the live demo link correctly. You can do it from Settings => Pages